obsei/analyzer/ner_analyzer.py

```diff
@@ -25,17 +25,15 @@ class NERAnalyzer(BaseAnalyzer):
     model_name_or_path: str
     tokenizer_name: Optional[str] = None
     grouped_entities: Optional[bool] = True
+    ner_config : BaseAnalyzerConfig = BaseAnalyzerConfig(batch_size=1)
 
     def __init__(self, **data: Any):
         super().__init__(**data)
 
         model = AutoModelForTokenClassification.from_pretrained(self.model_name_or_path)
-        if self.tokenizer_name:
-            tokenizer = AutoTokenizer.from_pretrained(
-                self.tokenizer_name, use_fast=True
-            )
-        else:
-            tokenizer = None
+        tokenizer = AutoTokenizer.from_pretrained(self.tokenizer_name if self.tokenizer_name else self.model_name_or_path
```

Comment:
please run [black formatting](https://github.com/lalitpagaria/obsei/blob/master/CONTRIBUTING.md#submitting-a-pull-request) on this code.